Src/Support/GoogleApis.Auth.NetStandard/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -0,0 +1,274 @@
+﻿/*
```

Comment:
Why do we need all these new files? Wouldn't it be simpler to put them in the shared folder and just use conditional compilation? I don't like the idea of changes being duplicated.